src/sections/Projects/Sistent/components/divider/index.js

```diff
@@ -0,0 +1,395 @@
+              separate items or components side by side, such as in a flex
+              container or sidebar. This orientation helps to create visual
+              breaks between closely related elements, enhancing readability and
+              organization. A vertical divider can be particularly effective in
+              navigation menus or grouped items where you want to maintain a
+              compact layout.
+            </p>
+          </div>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Box
+                sx={{
+                  display: "flex",
+                  alignItems: "center",
+                  border: "1px solid",
+                  borderColor: "divider",
+                  borderRadius: 1,
+                  bgcolor: "background.paper",
+                  color: "text.secondary",
+                  "& svg": {
+                    m: 1,
+                  },
+                }}
+              >
+                <FormatAlignLeftIcon />
+                <FormatAlignCenterIcon />
+                <FormatAlignRightIcon />
+                <Divider orientation="vertical" flexItem />
+                <FormatBoldIcon />
+              </Box>
+            </SistentThemeProvider>
+          </Row>
+          <a id="With children">
+            <h3>4. With children</h3>
+          </a>
+          <p>
+            Use the <code>textAlign</code> prop to align elements that are
+            wrapped by the Divider.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Root>
+                {content}
+                <Divider>CENTER</Divider>
+                {content}
+                <Divider textAlign="left">LEFT</Divider>
+                {content}
+                <Divider textAlign="right">RIGHT</Divider>
+                {content}
+                <Divider>
+                  <Chip label="Chip" size="small" />
+                </Divider>
+                {content}
+              </Root>
+            </SistentThemeProvider>
+          </Row>
+          <a id="Accessibility">
+            <h3>Accessibility</h3>
+          </a>
+          <p>
+            Due to its implicit role of separator, the Divider, which is a
+            <code>`hr`</code> element, will be announced by screen readers as a
+            "Horizontal Splitter" (or vertical, if you're using the orientation
+            prop). If you're using it as a purely stylistic element, we
+            recommend setting aria-hidden="true" which will make screen readers
+            bypass it.
+          </p>
+          <br />
+          <p>
+            If you're using the Divider to wrap other elements, such as text or
+            chips, we recommend changing its rendered element to a plain
```

Comment:
curious if a new contributor reading this doc will know what chips are? Idk if a link or referencing it as a component will make this more clear?